prime-dotnet-webapi/Configuration/AgreementVersionConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+                    new AgreementVersion { Id = 2,  AgreementType = AgreementType.RegulatedUserTOA,              Text =  "ru-access-terms-v1.html", EffectiveDate = SEEDING_DATE,                                CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 3,  AgreementType = AgreementType.OboTOA,                        Text = "obo-access-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-03-05 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 4,  AgreementType = AgreementType.RegulatedUserTOA,              Text =  "ru-access-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-03-05 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 5,  AgreementType = AgreementType.OboTOA,                        Text = "obo-access-terms-v3.html", EffectiveDate = DateTimeOffset.Parse("2020-03-10 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 6,  AgreementType = AgreementType.RegulatedUserTOA,              Text =  "ru-access-terms-v3.html", EffectiveDate = DateTimeOffset.Parse("2020-05-07 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 7,  AgreementType = AgreementType.OboTOA,                        Text = "obo-access-terms-v4.html", EffectiveDate = DateTimeOffset.Parse("2020-05-07 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 8,  AgreementType = AgreementType.RegulatedUserTOA,              Text =  "ru-access-terms-v4.html", EffectiveDate = DateTimeOffset.Parse("2020-06-03 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 9,  AgreementType = AgreementType.CommunityPharmacistTOA,        Text =  "com-pharm-terms-v1.html", EffectiveDate = SEEDING_DATE,                                CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 10, AgreementType = AgreementType.CommunityPharmacistTOA,        Text =  "com-pharm-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-08-28 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new AgreementVersion { Id = 11, AgreementType = AgreementType.CommunityPracticeOrgAgreement, Text = "com-practice-org-v1.html", EffectiveDate = SEEDING_DATE,                                CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
We need to add the Community Pharmacist Org Agreement as well.

Reply:
💯